config/filesystems.php

```diff
@@ -61,20 +87,6 @@
 			],
 		],
 
-		// This is an example how the "images" disk can be hosted on an AWS S3
-		// ATTENTION: This is NOT supported yet!!!
-		// This is only a placeholder/reminder for the future
-		/*
-		'images' => [
-			'driver' => 's3',
-			'key' => env('AWS_ACCESS_KEY_ID'),
-			'secret' => env('AWS_SECRET_ACCESS_KEY'),
-			'region' => env('AWS_DEFAULT_REGION', 'us-east-1'),
-			'bucket' => env('AWS_BUCKET'),
-			'url' => env('AWS_URL'),
-			'endpoint' => env('AWS_ENDPOINT'),
-		],*/
-
```

Comment:
Why do you add unhelpful default examples by the framework but throw out the only example which might eventually turn out to be helpful?

Reply:
Fixed in [bb29f73](https://github.com/LycheeOrg/Lychee/pull/1469/commits/bb29f730217eb210ec0f538355080e183a74fe0c)